### ISSUE TITLE
markdown on mobile has no keyboard

### ISSUE THREAD

**Issue by warp9pnt9** [jbendtsen/website#1]
Not quite sure why, but on my mobile browser (Brave, Android 12), the keyboard doesn't pop up when clicking in the input text field of the markdown editor.  Thought you might be interested to know.

**Comment by jbendtsen** [comment]
Yeah, the text field is actually an HTML canvas with some Javascript that listens for keyboard and mouse input, then renders a window's worth of text to the canvas. I haven't yet figured out a way to encourage the browser to bring up a virtual keyboard when the canvas has focus. Cursory research indicates that the only way to do that is to use either a textarea element instead or set contenteditable to true, which cannot be applied to canvas elements.
